tests/e2e/scripts/hermes_bootstrap.sh

```diff
@@ -57,7 +58,8 @@ account_prefix = 'osmo'
 key_name = 'val01-osmosis-b'
 store_prefix = 'ibc'
 max_gas = 6000000
-gas_price = { price = 0.000, denom = 'uosmo' }
+default_gas = 400000
```

Comment:
Why are we putting default gas? Won't we get out of gas errors routinely?

Reply:
AFAIU, this is what the relayer assumes for calculating fees when estimation fails ref: https://github.com/informalsystems/hermes/issues/1457

We use the value of 40k in the [tx arguments](https://github.com/osmosis-labs/osmosis/blob/b7d5fbddecf1df4743de3088c0a945b971f3a3e2/tests/e2e/containers/containers.go#L29) as well so I just reused it.

Reply:
Btw, the only remaining problem is the geometric twap failing for reasons I cannot currently explain. Please let me know if something immediately comes to mind as the solution.

I'm going to keep investigating in the meantime